### PR TITLE
Swagger search

### PR DESCRIPTION
**Pull request by BobZcao**
Please let me if I need to change anything.

### CODE REVIEW

**Review by xynazog — CHANGES_REQUESTED** [review]
The display is like this
```
{
Search results for keyword "iPhone" in wishlists with user ID "user3": [
{
Results from wishlist with ID: 3: [
{
description: "Product (Red) iPhone 7 Plus 256 GB",
item_id: "a12"
}
]
}
]
}
```
Your search shows
```
[
  {
    "wishlist_item_id": {
      "item_description": "string",
      "item_id": "string"
    }
  }
]
```

**Comment by BobZcao** [comment]
I see, I will fix it, thank you @xynazog

File: app/server.py [hunk 1]
```diff
@@ -737,19 +737,26 @@ def search_wishlists():
...
+                  type: object
+                  properties:
+                    item_id:
+                      type: string
+                      description: ID of the item matching
+                    item_description:
+                      type: string
+                      description: Description of the item
+      400:
+        description: userid is missing
```
Comment (pinocchiochang) [inline app/server.py]: Please make sure the format is consistent.

**Review by pinocchiochang — CHANGES_REQUESTED** [review]
As comment.

**Comment by BobZcao** [comment]
palak will take care of the colon. I set the module schema manually because it is only for documentation instead of the real database.

**Review by xynazog — COMMENTED** [review]
Looks great! Can you just remove the keywords like iPhone and all? And the example tags?
It would help with the consistency throughout the docs. Approved as soon as it's done 👍

**Review by palakbhasin — CHANGES_REQUESTED** [review]
In search, query is optional and userid is required.

**Comment by BobZcao** [comment]
fixed errors according to your suggestions

**Review by palakbhasin — APPROVED** [review]

File: app/server.py [hunk 2]
```diff
@@ -732,24 +732,36 @@ def search_wishlists():
...
+                        type: object
+                        properties:
+                          item_id:
+                            type: string
+                            description: ID of the item matching
+                          item_description:
+                            type: string
+                            description: Description of the item
+      400:
+        description: userid is missing
```
Comment (pinocchiochang) [inline app/server.py:764]: The format is chaos. I will fix tomorrow.

**Review by pinocchiochang — APPROVED** [review]
Just the format issue.

**Pull request merged**